src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTxWithDoubleSpendSpecification.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    public partial class SendingTransactionWithDoubleSpend
+    {
+        [Fact]
+        public void sending_transaction_with_double_spend()
```

Comment:
ok yes this is correct I would like to ask you make one more tests similar to this one that checks the double spent is confirmed.

So create the same test but mine the trx in to a block.
Should be just one more line of code and a different error message

Reply:
Done

Reply:
I now check it is mined and removed from mempool on the receiving node

When trying to double spend MempoolValidator throws error an checking the coinview but handles this internally, doesn't populate errors collection or propagate.  
Note this is why I don't call SendTransaction as it wraps AcceptToMemoryPool and will swallow error result